src/System.Diagnostics.TraceSource/tests/TraceEventCacheClassTests.cs

```diff
@@ -53,7 +53,6 @@ public void CallstackTest()
         {
             var cache = new TraceEventCache();
             Assert.Contains("at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)", cache.Callstack);
-            Assert.Contains("at System.Environment.get_StackTrace()", cache.Callstack);
```

Comment:
I did this because it was causing test failures in various builds.  Now that I removed the line, the remaining line is causing some of the other builds to fail, as in the Unix release builds, the entire string coming out is "at Internal.Runtime.Augments.EnvironmentAugments.get_StackTrace()".  I'm just going to disable the test for now against the issue you mention.